lib/apimatic-core/logger/default_logger.rb

```diff
@@ -28,12 +28,11 @@ def log(level, message, params = {})
     end
   end
 
-  def format_log_message(severity, datetime, _progname, msg)
-    "#{{
-      timestamp: datetime.strftime('%Y-%m-%dT%H:%M:%S.%6N'),
-      level: severity.ljust(5),
-      msg: msg
-    }}\n"
+  def format_log_message(severity, _datetime, _progname, msg)
+    "#{
```

Comment:
Question: Did you verify the output of the console logger?

Reply:
yes, the json structure will be removed as of now.
![image](https://github.com/apimatic/core-lib-ruby/assets/80243792/0588ec68-0418-4442-a89b-5c212ca13e2a)

Reply:
INFO Request POST http://localhost:3000/body/additionalModelProperties application/json
INFO Request headers {"Content-Type"=>"application/json", "accept"=>"application/json"}
INFO Request body {"name":"name2","field":"field4","address":"address8","Job":{"company":"company8"}}
INFO Response 400 449 application/json; charset=utf-8
INFO Response headers {"x-powered-by"=>"Express", "access-control-allow-origin"=>"*", "content-type"=>"application/json; charset=utf-8", "content-length"=>"449", "etag"=>"W/\"1c1-ROm4bD0y+jVMQasKLYtlzWF3ceY\"", "date"=>"Fri, 10 May 2024 07:00:24 GMT", "connection"=>"keep-alive", "keep-alive"=>"timeout=5"}
INFO Response body {"passed":false,"message":"Failed validation.","input":{"path":"/body/additionalModelProperties","query":{},"headers":{"content-type":"application/json","accept":"application/json","accept-encoding":"gzip,deflate","user-agent":"Ruby","connection":"keep-alive","keep-alive":"30","host":"localhost:3000","content-length":"83"},"method":"POST","body":{"name":"name2","field":"field4","address":"address8","Job":{"company":"company8"}},"uploadCount":0}}